app/src/main/java/nodomain/freeyourgadget/gadgetbridge/externalevents/NotificationListener.java

```diff
@@ -638,17 +653,14 @@ public void onNotificationRemoved(StatusBarNotification sbn) {
                 GBApplication.deviceService().onSetCallState(callSpec);
             }
         }
-        // FIXME: DISABLED for now
-        /*
         if (shouldIgnore(sbn))
             return;
 
         Prefs prefs = GBApplication.getPrefs();
-        if (prefs.getBoolean("autoremove_notifications", false)) {
+        if (prefs.getBoolean("autoremove_notifications", true)) {
```

Comment:
Maybe @ashimokawa has something to say about this default value change?

Reply:
for now that was for testing anyway, i only changed that since it was not available in the settings to change...